securedrop/requirements/python3/securedrop-app-code-requirements.txt

```diff
@@ -226,20 +265,22 @@ python-editor==1.0.3 \
 pytz==2017.3 \
     --hash=sha256:c41c62827ce9cafacd6f2f7018e4f83a6f1986e87bfd000b8cfbd4ab5da95f1a \
     --hash=sha256:fae4cffc040921b8a2d60c6cf0b5d662c1190fe54d718271db4eb17d44a185b7
-    # via babel
+    # via
+    #   babel
+    #   flask-babel
 qrcode==5.3 \
     --hash=sha256:4115ccee832620df16b659d4653568331015c718a754855caf5930805d76924e \
     --hash=sha256:60222a612b83231ed99e6cb36e55311227c395d0d0f62e41bb51ebbb84a9a22b
     # via -r requirements/python3/securedrop-app-code-requirements.in
-redis==3.3.6 \
-    --hash=sha256:45682ecf226c7611efe731974c4fa3390170ba045b9cdb26f0051114a5c2a68b \
-    --hash=sha256:f2609a85e5f37f489ba3b5652e1175dc3711c4d7a7818c4f657615810afd23df
+redis==3.5.3 \
```

Comment:
Reviewed: https://github.com/freedomofpress/securedrop-debian-packaging/wiki/redis-3.3.6-to-3.5.3